client/src/pages/CoursePage.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Link } from "react-router-dom"; // Import Link for internal navigation
+
+const CoursePage = () => {
```

Comment:
Call this `StudentCourseView` it is more descriptive of who is actually viewing these courses. also add more functionality as to getting all the options of courses and displaying that to the user. They need a way to know what are the option of courses they can choose from